src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -67,7 +67,7 @@ void CopyPasteManager::CopyToClipboard(String ^ stringToCopy)
     // Copy the string to the clipboard
     auto dataPackage = ref new DataPackage();
     dataPackage->SetText(stringToCopy);
-    Clipboard::SetContent(dataPackage);
+    Clipboard::SetContentWithOptions(dataPackage, ref new ClipboardContentOptions());
```

Comment:
Maybe we can simply give it a `nullptr` to use the built-in default option?

Reply:
Agree, nullptr will use the default options, which is more straightforward.